monitoring/src/config.ts

```diff
@@ -15,104 +15,104 @@ import { ConfigBuilder } from './utils/config-builder/config-builder';
-	stage: STAGES.CODE,
-	jurisdiction: JURISDICTIONS.TCFV2,
-	frontUrl: 'https://m.code.dev-theguardian.com',
-	articleUrl:
-		'https://m.code.dev-theguardian.com/food/2020/dec/16/how-to-make-the-perfect-vegetarian-sausage-rolls-recipe-felicity-cloake',
-	iframeDomain: 'https://sourcepoint.theguardian.com',
-	debugMode: debugMode,
-	checkFunction: mainCheckTcfV2,
-};
-
-const ConfigTcfv2Local: Config = {
-	stage: STAGES.LOCAL,
-	jurisdiction: JURISDICTIONS.TCFV2,
-	frontUrl: 'https://m.code.dev-theguardian.com',
-	articleUrl:
-		'https://m.code.dev-theguardian.com/food/2020/dec/16/how-to-make-the-perfect-vegetarian-sausage-rolls-recipe-felicity-cloake',
-	iframeDomain: 'https://sourcepoint.theguardian.com',
-	debugMode: debugMode,
-	checkFunction: mainCheckTcfV2,
-};
-
-const ConfigCCPAProd: Config = {
-	stage: STAGES.PROD,
-	jurisdiction: JURISDICTIONS.CCPA,
-	frontUrl: 'https://www.theguardian.com/us',
-	articleUrl:
-		'https://www.theguardian.com/food/2020/dec/16/how-to-make-the-perfect-vegetarian-sausage-rolls-recipe-felicity-cloake',
-	iframeDomain: 'https://sourcepoint.theguardian.com',
-	debugMode: debugMode,
-	checkFunction: mainCheckCCPA,
-};
-
-const ConfigCCPACode: Config = {
-	stage: STAGES.CODE,
-	jurisdiction: JURISDICTIONS.CCPA,
-	frontUrl: 'https://m.code.dev-theguardian.com/us',
-	articleUrl:
-		'https://m.code.dev-theguardian.com/food/2020/dec/16/how-to-make-the-perfect-vegetarian-sausage-rolls-recipe-felicity-cloake',
-	iframeDomain: 'https://sourcepoint.theguardian.com',
-	debugMode: debugMode,
-	checkFunction: mainCheckCCPA,
-};
-
-const ConfigCCPALocal: Config = {
-	stage: STAGES.LOCAL,
-	jurisdiction: JURISDICTIONS.CCPA,
-	frontUrl: 'https://m.code.dev-theguardian.com/us',
-	articleUrl:
-		'https://m.code.dev-theguardian.com/food/2020/dec/16/how-to-make-the-perfect-vegetarian-sausage-rolls-recipe-felicity-cloake',
-	iframeDomain: 'https://sourcepoint.theguardian.com',
-	debugMode: debugMode,
-	checkFunction: mainCheckCCPA,
-};
-
-const ConfigAusProd: Config = {
-	stage: STAGES.PROD,
-	jurisdiction: JURISDICTIONS.AUS,
-	frontUrl: 'https://www.theguardian.com/au',
-	articleUrl:
-		'https://www.theguardian.com/food/2020/dec/16/how-to-make-the-perfect-vegetarian-sausage-rolls-recipe-felicity-cloake',
-	iframeDomain: 'https://sourcepoint.theguardian.com',
-	debugMode: debugMode,
-	checkFunction: mainCheckAus,
-};
-
-const ConfigAusCode: Config = {
-	stage: STAGES.CODE,
-	jurisdiction: JURISDICTIONS.AUS,
-	frontUrl: 'https://m.code.dev-theguardian.com/au',
-	articleUrl:
-		'https://m.code.dev-theguardian.com/food/2020/dec/16/how-to-make-the-perfect-vegetarian-sausage-rolls-recipe-felicity-cloake',
-	iframeDomain: 'https://sourcepoint.theguardian.com',
-	debugMode: debugMode,
-	checkFunction: mainCheckAus,
-};
-
-const availableEnvConfig = [
-	ConfigTcfv2Prod,
-	ConfigTcfv2Code,
-	ConfigCCPAProd,
-	ConfigCCPACode,
-	ConfigAusProd,
-	ConfigAusCode,
-	ConfigCCPALocal,
-	ConfigTcfv2Local,
-];
+// const ConfigTcfv2Prod: Config = {
```

Comment:
Should this all be commented out?

Reply:
I understand this is superceded configuration code and have removed it after running all tests.